window.c

```diff
@@ -133,6 +134,25 @@ void win_init(win_t *win)
 	f = win_res(db, RES_CLASS ".font", "monospace-8");
 	win_init_font(e, f);
 
+	const char *title_prefix = win_res(db, RES_CLASS ".title.prefix", NULL);
+	const char *title_suffixmode = win_res(db, RES_CLASS ".title.suffix", NULL);
```

Comment:
Shouldn't we register .title.suffixmode here?

Reply:
Probably. Now that I am editing the code, I kept thinking that.